src/interface/src/app/shared/form-message-box/form-message-box.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnInit } from '@angular/core';
+//import { FormMessageType } from '../../types/data.types';
+
+@Component({
+  selector: 'app-form-message',
+  templateUrl: './form-message-box.component.html',
+  styleUrls: ['./form-message-box.component.scss'],
+})
+export class FormMessageBoxComponent implements OnInit {
+  @Input() title!: string;
+  @Input() message?: string;
+  @Input() showMessage: boolean = false;
+  @Input() messageType: string = '';
```

Comment:
I was considering an enum for this instead...

Reply:
yeah I agree an enum or even a type will help here.